tests/install_renku.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+target_virtualenv_version=20.0.29
+installed_virtualenv_version=`virtualenv --version`
+echo "Installed virtualenv version: $installed_virtualenv_version"
+
+if [[ ! "$installed_virtualenv_version" == *"$target_virtualenv_version" ]]; then
+  echo "WARNING: Incompatible version of virtualenv installed: $installed_virtualenv_version . Replacing with version $target_virtualenv_version"
+  python3 -m pip install virtualenv==$target_virtualenv_version
+fi
+
+virtualenv venv
+source ./venv/bin/activate
+pip install 'renku==%s'
```

Comment:
you're installing `renku` inside the `FlightsTutorial` code - so what is the purpose of installing it here?

Reply:
I think previously we were installing it globally so it made sense to do it here but now i think this script should just be used to check the virtualenv version. I'll take out the lines which set up a virtualenv and install renku